internal/host/hostcommands/instruction_manager.go

```diff
@@ -136,13 +144,19 @@ func (i *InstructionManager) GetNextSteps(ctx context.Context, host *models.Host
 			if steps == nil {
 				continue
 			}
+			// Creating StepID when needed and filtering out disabled steps
+			enabledSteps := []*models.Step{}
 			for _, step := range steps {
+				if i.isStepDisabled(step.StepType) {
+					log.WithField("disabledStepsMap", i.disabledStepsMap).Infof("Step '%v' is disabled. Will not include it in instructions", step.StepType)
```

Comment:
Why print the whole map?

Reply:
Leftover from debugging. will be addressed in https://github.com/openshift/assisted-service/pull/2276